lms/djangoapps/courseware/module_render.py

```diff
@@ -934,6 +939,29 @@ def handle_xblock_callback(request, course_id, usage_id, handler, suffix=None):
         return _invoke_xblock_handler(request, course_id, usage_id, handler, suffix, course=course)
 
 
+# TODO: deprecated (XblockCallbackView is deprecated in favour of handle_xblock_callback as a function based view)
```

Comment:
`XblockCallbackView` is not deprecated. We contributed it to the edx-solutions ficus version: https://github.com/edx-solutions/edx-platform/pull/856
And we are trying to get it merged into upstream master (Hawthorn) too, just waiting on edX code review: https://github.com/edx/edx-platform/pull/15940

Reply:
I think our commit got left out of this rebase somehow. I recommend cherry-picking it, since it includes all the necessary test fixes. Or cherry-pick the newer upstream version and backport it.

Reply:
@bradenmacdonald yes you are right. we lost this commit during rebase process. I have added this commit in our ginkgo-release branch and things are looking good now. i will push my code tomorrow. Thank you for your suggestion.